### PR TITLE
Added a graph print feature closes #19

### LINKED ISSUE #19: Add a graph printing feature

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -10,7 +10,7 @@ scalaVersion := "2.11.8"
 
 lazy val versionSettings = Seq(
   // Upcoming release, or current if we're on the master branch
-  git.baseVersion := "0.8",
+  git.baseVersion := "8",
```
Comment (geoffjentry) [inline build.sbt]: As much as I was an advocate for this on Cromwell I was just wondering about this for our other tools, I came up with arguments on both sides. We should discuss.

File: src/main/scala/wdltool/Main.scala [hunk 2]
```diff
@@ -72,6 +74,76 @@ object Main extends App {
     }
   }
 
+  def graph(args: Seq[String]): Termination = {
```
Comment (geoffjentry) [inline src/main/scala/wdltool/Main.scala:77]: I'd prefer to see the graph logic out of Main (yes, the other stuff is here too but the other stuff isn't as long) but don't care that much if you decide to not bother

**Comment by mcovarr** [comment]
very cool :+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/wdltool/pull-request/18/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/wdltool)

**Comment by geoffjentry** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/wdltool/pull-request/18/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/wdltool)

**Pull request merged**